src/libs/SidebarUtils.js

```diff
@@ -326,7 +327,17 @@ function getOptionData(reportID) {
     }
 
     const reportName = ReportUtils.getReportName(report);
+
     result.text = reportName;
+    if (result.isThread) {
```

Comment:
Similar comment to clean this up as suggested [here](https://github.com/Expensify/App/pull/18522#discussion_r1190391045)